infrastructure/environments/dplplat01/configuration/loki/loki-values.template.yaml

```diff
@@ -75,3 +156,26 @@ monitoring:
         operator: Exists
       - key: noderole.dplplatform
         operator: Exists
+  selfMonitoring:
+    grafanaAgent:
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+      # grafana operator agent does not support affinity in this version of the chart - should be fixed later
```

Comment:
why is this commented out?

Reply:
uh - see comment I wrote.
> grafana operator agent does not support affinity in this version of the chart - should be fixed later

Reply:
theres no support for setting affinity on the pod